app/src/main/res/values-de/strings.xml

```diff
@@ -606,7 +606,7 @@
     <string name="video_options">Videooptionen</string>
     <string name="change_pins">Pins ändern</string>
     <string name="decide_which_buttons_should_be_pinned">Entscheiden Sie, welche Tasten angeheftet werden sollen</string>
```

Comment:
Tasten should be replaced with Schaltflächen due to the fact that Tasten are tactile switches and buttons are Schaltflächen.